tests.py

```diff
@@ -23,6 +23,10 @@
 def run(extra_args):
     from nose.plugins import multiprocess
 
+    env = {"NOSE_WITH_COVERAGE": 1,
+           'NOSE_COVER_PACKAGE': 'matplotlib',
+           'NOSE_COVER_HTML': 1}
```

Comment:
Since code coverage can slow down runtimes, and it's not compatible with multiprocessing tests, we should probably have this be an option controlled by a commandline flag.  Then on Travis-CI, we can do this just on one configuration (probably Python 3.5), and keep all the others running with multiple processes.